provider/event/events.go

```diff
@@ -53,5 +54,16 @@ type ClusterDeployment struct {
 	Status  ClusterDeploymentStatus
 }
 
-// Empty type used as a marker to indicate _any_ lease should be withdrawn now
-type LeaseWithdrawNow struct{}
+// LeaseWithdraw Empty type used as a marker to indicate specified lease should be withdrawn now
+type LeaseWithdraw struct {
```

Comment:
Might be a Github display thing, but where is this type created at? I see some code checking if it receives it, but not creating it.

Reply:
Nevermind, github decided not to show diffs by default now :(

Reply:
`startWithdraw` function in `balance_checker.go`